afsbirez/settings/dev.py

```diff
@@ -21,4 +22,9 @@
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-INSTALLED_APPS.append('django_extensions')
\ No newline at end of file
+INSTALLED_APPS.append('django_extensions')
+
+REST_PROXY = {
```

Comment:
This block is in both the dev and production configuration files. Should it be moved to the base configuration.

Alternatively, we could remove the credentials.py dependency from dev.py by replacing the actual key with `DEMO_KEY`.